### ISSUE TITLE
Problem with RxAtomic when trying to run iOS acc demo

### ISSUE THREAD

**Issue by joezein** [polarofficial/polar-ble-sdk#64]
```
dyld: Library not loaded: @rpath/RxAtomic.framework/RxAtomic
  Referenced from: /private/var/containers/Bundle/Application/B9348436-9CDB-49A5-A180-33C7C58D2DBD/PolarSDK-ACC-Demo.app/PolarSDK-ACC-Demo
  Reason: no suitable image found.  Did find:
	/private/var/containers/Bundle/Application/B9348436-9CDB-49A5-A180-33C7C58D2DBD/PolarSDK-ACC-Demo.app/Frameworks/RxAtomic.framework/RxAtomic: code signature invalid for '/private/var/containers/Bundle/Application/B9348436-9CDB-49A5-A180-33C7C58D2DBD/PolarSDK-ACC-Demo.app/Frameworks/RxAtomic.framework/RxAtomic' 
```

Hi, I am getting this error every time I try to run this on my device. App on my phone stays stuck on LaunchScreen. 

Can you please help me make it work?

I am trying to run the iOS Acc demo

**Comment by joezein** [comment]
![Screen Shot 2020-03-10 at 1 15 09 AM](https://user-images.githubusercontent.com/58489838/76281945-b2901b00-626c-11ea-809a-9bf46ffc8bcd.png)

This is what I get

**Comment by erkki-silvola** [comment]
Hi,

I have not checked the ACC demo for a long time, so it might be outdated... I will check it this week, mean while try the exampleApp which should work just fine

**Comment by joezein** [comment]
I  still get the  same issue for example. Failed to register bundle+ No profile

**Comment by erkki-silvola** [comment]
Try to change the bundle identifier

**Comment by pajama** [comment]
I was able to get it to build by removing the rxatomic framework on the general tab. RxSwift is the replacement to atomic.

![image](https://user-images.githubusercontent.com/2224105/77782786-0d6b9580-7015-11ea-87e5-27e016b02420.png)

I then had to add the bluetooth usage description to the info.plist for it to not crash on load.

However for any of the demos/examples that support ACC I get the error 
`PolarBleSdk.InvalidArgument`
